packages/react-strict-dom/src/native/stylex/index.js

```diff
@@ -300,7 +302,11 @@ function processStyle<S: { +[string]: mixed }>(style: S): S {
           if (propName === 'color') {
             result['placeholderTextColor'] = styleValue[propName];
           } else {
-            warnMsg(`unsupported "::placeholder" style property "${propName}"`);
+            if (__DEV__) {
+              warnMsg(
+                `unsupported "::placeholder" style property "${propName}"`
+              );
+            }
```

Comment:
The "small" benchmark gets a little slower when this `warnMsg` or (less so) when the one for `boxShadow` is commented out...even if the `warnMsg` function is a no-op. Given this code is never reached for those benchmarks, it must just be a bizarre artifact that doesn't represent any real-world issue.

Reply:
I'm not profiling with a benchmark. I was profiling a real world scenario. Is it possible that my real world scenario just uses more style code that triggers this warning more often?

Reply:
No. The benchmark that includes this style gets faster with the logs removed - as expected. The bizaree thing is that a different benchmark with 2 basic styles show a regression, but it never even reaches this code. As mentioned above, the regression only occurred when the log function is removed but not if the body of the log function itself is commented out - so it's unrelated to logging itself and doesn't really make sense to me.